backend/services/userService.js

```diff
@@ -20,6 +20,15 @@ class UserService {
       throw error;
     }
   }
+
+  static async getUser(email) {
+    try {
+      const user = await database.Users.findOne({ where: { email } });
```

Comment:
Let's do as we did in Products Service.js File.

import database from '../database/models';

const { Users } = database;

then we can use just Users.findOne(...

Reply:
thanks for suggesting the changes. You can check their implementation in an updated file.